packages/client/components/TaskFooterIntegrateMenuList.tsx

```diff
@@ -70,13 +70,13 @@ const Label = styled('div')({
 const getValue = (
   item: NonNullable<TaskFooterIntegrateMenuList_suggestedIntegrations['items']>[0]
 ) => {
-  const jiraItemName = item?.remoteProject?.name ?? ''
+  const jiraItemName = item?.projectKey ?? ''
```

Comment:
`remoteProject` is undefined because `teamId` is undefined [here](https://github.com/ParabolInc/parabol/blob/8409e5ee0ab86c9effcb616dd82cdf30053c81ee/packages/server/graphql/types/SuggestedIntegrationJira.ts#L33). I'm not sure why the source doesn't have the `teamId`, as `SuggestedIntegration` belongs to the `TeamMember` which must have the `teamId`. 

As I'm away tomorrow & Friday, I thought it would be best to share this PR now rather than wait until I get to the bottom of this, which would probably be on Monday. 

We can still filter by `projectKey` as this is what we're displaying in the dropdown.